x-pack/plugins/apm/public/components/app/service_map/use_ref_dimensions.ts

```diff
@@ -13,7 +13,7 @@ export function useRefDimensions() {
   const windowHeight = useWindowSize().height;
 
   if (!ref.current) {
-    return { ref, width: 0, height: 0 };
+    return { ref, width: 0, height: windowHeight };
```

Comment:
I'm not fully sure about this, but was the only solution I came up with, in case that the `ref.current` (service map div) is not there

Reply:
Discussed during tech session